### PR TITLE
Add texts extracted from slides for accessibility

### PR DESCRIPTION
**Pull request by gustavotrott**
This PR:

- Get the text of the current slide from txt file.
- Set the attribute `aria-describedby` of presentation component with the text of current slide.

_This was requested in #6692_

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/presentation/component.jsx [hunk 1]
```diff
@@ -231,7 +243,10 @@ class PresentationArea extends Component {
     return (
       <div
         style={svgDimensions}
+        aria-label="Slide Content"
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/component.jsx]: Should we localize?

File: bigbluebutton-html5/imports/api/presentations/server/modifiers/addPresentation.js [hunk 2]
```diff
@@ -6,10 +7,20 @@ import flat from 'flat';
 import addSlide from '/imports/api/slides/server/modifiers/addSlide';
 import setCurrentPresentation from './setCurrentPresentation';
 
+const getSlideText = async (url) => {
+  const file = await HTTP.get(url);
+
+  return file;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/presentations/server/modifiers/addPresentation.js]: What happens if the slide has no [obtainable] text? - i.e. image slide

**Comment by capilkey** [comment]
I think the standard way to interrupt the screen reader is Ctrl. I'm not sure if it's the same with all screen readers, but the user will have control of it.

File: bigbluebutton-html5/imports/ui/components/presentation/component.jsx [hunk 3]
```diff
@@ -231,7 +243,10 @@ class PresentationArea extends Component {
...
+        aria-describedby="currentSlideText"
       >
+        <p id="currentSlideText" hidden>{slideContent}</p>
```
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/ui/components/presentation/component.jsx]: Using `hidden` like this also hides the content from NVDA, I'd suggest adding a `className` to hide it visually.
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/nav-bar/recording-indicator/styles.scss#L4

File: bigbluebutton-html5/imports/ui/components/presentation/service.js [hunk 4]
```diff
@@ -23,15 +23,6 @@ const getCurrentSlide = (podId) => {
       presentationId: currentPresentation.id,
       current: true,
     },
-    {
-      fields: {
-        meetingId: 0,
-        thumbUri: 0,
-        swfUri: 0,
-        txtUri: 0,
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/service.js]: Should we not simply switch `txtUri: 0` to `1`? Looks like we could still exclude the other properties

Comment (gustavotrott) [reply bigbluebutton-html5/imports/ui/components/presentation/service.js]: I tried, but received this error:
`You cannot currently mix including and excluding fields`

File: bigbluebutton-html5/imports/ui/components/presentation/component.jsx (hunk 1 above)
Comment (gustavotrott) [reply bigbluebutton-html5/imports/ui/components/presentation/component.jsx]: I think so!

File: bigbluebutton-html5/imports/api/presentations/server/modifiers/addPresentation.js (hunk 2 above)
Comment (gustavotrott) [reply bigbluebutton-html5/imports/api/presentations/server/modifiers/addPresentation.js]: The file is generated even for slides without texts.
In this case the content is empty.

File: bigbluebutton-html5/imports/ui/components/presentation/component.jsx (hunk 3 above)
Comment (gustavotrott) [reply bigbluebutton-html5/imports/ui/components/presentation/component.jsx]: Done in last commit for this PR.

**Pull request merged**